server/public/model/utils_test.go

```diff
@@ -200,6 +201,89 @@ func TestMapJson(t *testing.T) {
 	require.LessOrEqual(t, len(rm2), 0, "make should be invalid")
 }
 
+func TestArrayFromJsonLimited(t *testing.T) {
+	t.Run("Successful parse", func(t *testing.T) {
+		ids := []string{NewId(), NewId(), NewId()}
+		b, _ := json.Marshal(ids)
+		a, err := ArrayFromJSONLimited(bytes.NewReader(b), 1000)
+		require.NoError(t, err)
+		require.Equal(t, len(ids), len(a))
+	})
+
+	t.Run("Empty Array", func(t *testing.T) {
+		ids := []string{}
+		b, _ := json.Marshal(ids)
+		a, err := ArrayFromJSONLimited(bytes.NewReader(b), 1000)
+		require.NoError(t, err)
+		require.Equal(t, 0, len(a))
+	})
+
+	t.Run("Error too large", func(t *testing.T) {
+		var ids []string
+		for i := 0; i <= 100; i++ {
+			ids = append(ids, NewId())
+		}
+		b, _ := json.Marshal(ids)
+		_, err := ArrayFromJSONLimited(bytes.NewReader(b), 1000)
+		require.Error(t, err)
```

Comment:
Wondering if we can also verify the error returned here? From what I can see in the docs, it's `io.EOF`. So something like `require.True(t, errors.Is(err, io.EOF))` should work.

Reply:
done